tools/deb/istio-start.sh

```diff
@@ -30,8 +30,8 @@ fi
 ISTIO_BIN_BASE=${ISTIO_BIN_BASE:-/usr/local/bin}
 ISTIO_LOG_DIR=${ISTIO_LOG_DIR:-/var/log/istio}
 ISTIO_CFG=${ISTIO_CFG:-/var/lib/istio}
-NS=${POD_NAMESPACE:-default}
-SVC=${ISTO_SERVICE:-rawvm}
+NS=${ISTIO_NAMESPACE:-default}
```

Comment:
AFAIK ISTIO_NAMESPACE is istio-system
POD_NAMESPACE is meant to mean the namespace where this VM (acting as a pod) should use (for whatever stats envoy is using)